terraspace/app/stacks/shared/auth.tf

```diff
@@ -98,55 +98,38 @@ EOF
 resource "aws_iam_policy" "s3_dotstorage_policy_read" {
```

Comment:
We could call it `s3_policy_read` now, because it's no longer strongly attached with dotstorage.
That would require state mv

Reply:
I'll leave this out for now.  Its a larger task than just this one resource.  `dotstorage` is used in multiple places, and we should have a cleanup task to resolve it.